tests/forcetorque/README.md

```diff
@@ -0,0 +1,8 @@
+## ForceTorque plugin Test
+
+~~~
+cd build
+export GZ_SIM_SYSTEM_PLUGIN_PATH=`pwd`
+cd tests/forcetorque
+./ForceTorqueTest
+~~~
```

Comment:
As mentioned in the comment to the PR as an whole, I suggest to remove this and just document in the README how to run tests.

```suggestion
```